jujugui/static/gui/src/app/app.js

```diff
@@ -1086,6 +1086,10 @@ YUI.add('juju-gui', function(Y) {
         // as it's visible but not functional.
         showEnvSwitcher = false;
       }
+      // If we're in sandbox we don't want to display the switcher.
+      if (this.get('sandbox')) {
+        showEnvSwitcher = false;
+      }
```

Comment:
Not that I think it makes a huge difference, but is there any reason this isn't merged with the if condition above? This is just one more condition for which the env switcher should not be shown.